src/Telemetry/Metering/Metrics.Generators/Metrics.Generators.csproj

```diff
@@ -3,19 +3,23 @@
   <PropertyGroup>
     <Description>Demonstrates how to emit application metrics.</Description>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>$(NetCoreTargetFrameworks)</TargetFrameworks>
-    <!-- This activates metrics report generator -->
-    <GenerateMetricsReport>true</GenerateMetricsReport>
-    <!-- Specify where the MetricReport.json file should be created -->
-    <MetricsReportOutputPath>$(OutDir)</MetricsReportOutputPath>
+
+    <!-- This property is used to control whether metrics report generator kicks in -->
+    <!--<GenerateMetricsReport>true</GenerateMetricsReport>-->
+
+    <!-- 
+        Optionally you can control where the "MetricsReport.json" file should be created.
+        If this value is changed, make sure to make a corresponding change in MetricsReportCheckerService.cs.
+     -->
+    <!--<MetricsReportOutputPath>$(ProjectDir)$(OutputPath)</MetricsReportOutputPath>-->
```

Comment:
This looks different to src/Compliance/AuditReports/AuditReports.csproj... Should these be alike?

Reply:
Yes, will update it